src/actions/__tests__/socialAuthActions.test.js

```diff
@@ -0,0 +1,71 @@
+/* eslint-disable comma-dangle */
```

Comment:
Why is this rule disabled for this file?

Reply:
I have fixed them